app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  has_secure_password
```

Comment:
So a user can have many payments?

Reply:
Yes apu. Added the validation. Thank you.